setup.py

```diff
@@ -167,6 +167,7 @@ def get_exts_for(name):
         "enum34; python_version<'3.4'",
         "funcsigs>=1.0.0; python_version=='2.7'",
         "typing; python_version<'3.5'",
+        "typing-extensions; python_version<'3.8'",
```

Comment:
is there a minimum required version we need for this to work?

Reply:
Yup typing.TypedDict was introduced in [python 3.8](https://www.python.org/dev/peps/pep-0589/). For older python versions we need typing-extensions.

Reply:
typing extensions 4.0 does not support python 2.7 (only python 3.0+) and typing extensions 3.7.2 does not have TypedDicts (we need `typing-extensions>=3.7.4,<4.0"`)